### ISSUE TITLE
Routing Table persistence

### ISSUE THREAD

**Issue by kumavis** [libp2p/js-libp2p-kad-dht#109]
Should the routing table be persisted? Providers currently are. Maybe if we don't have a ping mechanism, we want to avoid persisting the routingTable. Curious what the approach is here on what to persist and what not to.

**Comment by achingbrain** [comment]
Yes, see related issue: https://github.com/libp2p/go-libp2p-kad-dht/issues/254
